src/main/webapp/app/meta/meta.component.html

```diff
@@ -32,15 +32,7 @@ <h1><i class="fa fa-spinner fa-spin"></i></h1>
             </ngx-datatable-column>
             <ngx-datatable-column name="Status" [flexGrow]="1">
                 <ng-template ngx-datatable-cell-template let-row="row" let-value="value">
-                    <span
```

Comment:
How do you edit the status now?

Reply:
@zhx828 Select dropdown list will be shown on page directly.
![image](https://user-images.githubusercontent.com/14971266/59040700-3d0e9e80-8845-11e9-934f-d3a2f3e5236b.png)